.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -39,3 +20,15 @@ body:
       label: Actual behaviour
     validations:
       required: true
+  - type: markdown
+    attributes:
+      value: '## Environment'
+  - type: input
+    attributes:
+      label: Grafana k6 Studio version
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: OS
+      description: e.g. Windows 11, Arch Linux, macOS 15.3.1, etc.
```

Comment:
I moved the environment section to the bottom as steps to reproduce and expected/actual behavior seem more important